docs/flowcollector-flows-netobserv-io-v1beta2.adoc

```diff
@@ -276,36 +436,93 @@ Type::
 |===
 | Property | Type | Description
 
-| `cacheActiveTimeout`
-| `string`
-| `cacheActiveTimeout` is the max period during which the reporter aggregates flows before sending.
+| `disableAlerts`
+| `array (string)`
+| `disableAlerts` is a list of alerts that should be disabled.
+Possible values are: +
 
-| `cacheMaxFlows`
+`NetObservDroppedFlows`, which is triggered when the eBPF agent is dropping flows, such as when the BPF hashmap is full or the capacity limiter being triggered. +
+
+
+| `enable`
+| `boolean`
+| Set `enable` to `false` to disable eBPF agent metrics collection. It is enabled by default.
+
+| `server`
+| `object`
+| Metrics server endpoint configuration for Prometheus scraper
+
+|===
+== .spec.agent.ebpf.metrics.server
+Description::
++
+--
+Metrics server endpoint configuration for Prometheus scraper
+--
+
+Type::
+  `object`
+
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `port`
 | `integer`
-| `cacheMaxFlows` is the max number of flows in an aggregate; when reached, the reporter sends the flows.
+| The prometheus HTTP port
```

Comment:
```suggestion
| The Prometheus HTTP port
```

Reply:
I think Prometheus always be capitalized.

Reply:
Actually changing this to "The metrics server HTTP port" , that's more correct